src/env.rs

```diff
@@ -170,6 +171,29 @@ impl Rkv {
     }
 }
 
+/// Other environment methods.
+impl Rkv {
+    /// Flush the data buffers to disk.
```

Comment:
Great comments!

To minimize the confusion, could you add one more comment here to emphasize the manual sync could only be useful for environments opened with `MDB_NOSYNC`, `MDB_NOMETASYNC`, or `MDB_MAPASYNC`?